docs/releases/updating_ez_platform.md

```diff
@@ -242,6 +242,33 @@ These steps are only relevant for some releases:
     ALTER TABLE ezuser ALTER COLUMN password_hash TYPE VARCHAR(255);
     ​```
 
+??? note "v2.2: Change from UTF8 to UTF8MB4"
+
+    Since v2.2 the character set for MySQL/MariaDB database tables changes from `utf8` to `utf8mb4` to support 4-byte characters.
+
+    To apply this change, use the following database update script:
+
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql
+    ```
+
+    Be aware that these upgrade statements may fail due to index collisions.
+    This is because the indexes have been shortened, so duplicates may occur.
+    If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+
```

Comment:
After this upgrade script is run, one must change the character set (and collation) of database tables. This is described in https://github.com/ezsystems/developer-documentation/pull/241/files
_After successfully running those statements, change the character set of each table..._

Reply:
(I don't much like all the steps that the db admins will have to do manually, but I think it is the best/safest way.)